components/image/picker/src/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+
+class ImagePicker extends Component {
+  constructor (props) {
+    super()
+
+    const { images } = props
+    let filteredImages = []
+    let selected = -1
+
+    if (images.length > 0) {
+      filteredImages = images.map(image => {
+        if (typeof image.src !== 'undefined' && typeof image.thumb !== 'undefined') { return image }
```

Comment:
Why would you map and then filter when you can just filter ?
```
images.filter(img => img.thumb && img.src)
```
Use "truthy values" so you don't validate a 0 or "" or null as a value...

Reply:
Really good point!!